src/main/java/me/xemor/superheroes/skills/implementations/DamageModifierSkill.java

```diff
@@ -0,0 +1,100 @@
+package me.xemor.superheroes.skills.implementations;
+
+import me.xemor.superheroes.data.HeroHandler;
+import me.xemor.superheroes.skills.Skill;
+import me.xemor.superheroes.skills.skilldata.SkillData;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.entity.EntityDamageByEntityEvent;
+import org.bukkit.event.entity.EntityDamageEvent;
+import me.xemor.superheroes.skills.skilldata.DamageModifierData;
+
+import java.util.Collection;
+
+public class DamageModifierSkill extends SkillImplementation {
+    public DamageModifierSkill(HeroHandler heroHandler) {
+        super(heroHandler);
+    }
+
+    @EventHandler
+    public void onDamage(EntityDamageByEntityEvent event) {
+
+
+        double offence = event.getDamage();
+        double offencePrioity = Integer.MIN_VALUE;
+
+        double defence = event.getDamage();
+        double defencePriority = Integer.MIN_VALUE;
```

Comment:
This priority system is clever, but I'm not sure when it would ever be used. Does it default to having the defender have higher priority, ie calculate offense then calculate defense?

Reply:
It defaults to defender having higher priority. 

As for when it would be used, a possible situation might be a defense oriented hero who is intended to be stronger then a particular offense oriented hero for lore, roleplay or gameplay reasons.